.travis.yml

```diff
@@ -9,10 +9,9 @@ python:
   - 2.7
 
 env:
-  - ANSIBLE_INSTALL_VERSION=2.2.3.0
-  - ANSIBLE_INSTALL_VERSION=2.3.3.0
-  - ANSIBLE_INSTALL_VERSION=2.4.4.0
-  - ANSIBLE_INSTALL_VERSION=2.5.0
+  - ANSIBLE_INSTALL_VERSION=2.5.9
```

Comment:
Not quite the latest versions.

Reply:
Should be 2.5.14 and 2.6.13.

Reply:
Sorry :)

Reply:
Good catch.  My script that pulls the latest versions filters on the string value, not the int value :-(